test/functional/DDR_Test/src/j9vm/test/ddrext/junit/TestDDRExtensionGeneral.java

```diff
@@ -99,9 +99,8 @@ public void testWhatIsSetDepth() {
 	}
 
 	/*
-	 * Performs 2 tests: runs !classforname java/lang/Object first, then uses
-	 * the address of j9class to run !whatis runs !methodforname sleep first,
-	 * then uses the address of the j9method to run !what is
+	 * Performs test: runs !classforname j9vm/test/corehelper/CoreGen first, then uses
+	 * the address of j9class to run !whatis.
```

Comment:
Perhaps I should have been explicit: this should be paired with removing the commented-out test (lines 112-120).